packages/api/src/api/index.ts

```diff
@@ -1,7 +1,9 @@
 /* istanbul ignore file */
 import { Router } from 'express';
 import { health } from './health';
+import { users } from './users';
 
 export const api = Router();
 
 api.use('/health', health);
+api.use('/users', users);
```

Comment:
Add `newline` to end of file for linting purposes
```suggestion
api.use('/users', users);

```